Kconfig

```diff
@@ -181,3 +203,53 @@ config FIRMWARE_SECURE_VER
 	help
 	  Use specific anti-rollback secure version rather than the default 0.
 	  Range 0 - 127.
+
+choice
+	prompt "*** OTP provisioning command type ***"
```

Comment:
This `***` separator does not look very beautiful IMHO.

Reply:
![image](https://user-images.githubusercontent.com/16953829/176385355-5a773174-734a-4d9d-ae7a-07369c1de778.png)

Reply:
![image](https://user-images.githubusercontent.com/16953829/176385641-2a673b66-e588-4893-bab5-99228a7e6723.png)

Reply:
Ah, was me in fact :smile: